Add reminder not to remove user from .mailmap (#3716)
alphagov/govuk-developer-docs: source/manual/removing-a-user-from-puppet.html.md

1. Create another PR for Puppet that:
  - Removes the user manifest file
  - Removes the user from [Integration users][integration-users]

>   - DO NOT remove the user's email from the [.mailmap file][mailmap] if present - this allows use to keep a nicer git commit history once people have left.

Reviewer: A typo I think - here's what I think you meant:

```suggestion
  - DO NOT remove the user's email from the [.mailmap file][mailmap] if present. It allows the user to keep a nicer git commit history.
```

Author: Oh, I actually meant "this allows *us*", but your change is also good. Will fix.
